src/aui-form-builder/js/aui-form-builder-pages.js

```diff
@@ -194,6 +196,16 @@ A.FormBuilderPages = A.Base.create('form-builder-pages', A.Base, [], {
     },
 
     /**
+     * Fired after the `mode` attribute changes.
+     *
+     * @method _afterModeChange
+     * @protected
+     */
+    _afterModeChange: function() {
```

Comment:
_afterModeChange should go before _afterPagesQuantityChange in alphabetical order.